### PR TITLE
22.3 fb hl7 demographics

### PR DESCRIPTION
**Pull request by spamhurts**
#### Rationale
New process used to export demographics data to Orchard Harvest LISZ

#### Related Pull Requests
None

#### Changes
new code

### CODE REVIEW

File: snprc_ehr/resources/queries/study/DemographicsHL7Deleted.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+select
+       '0186B9BD-E5EF-406A-B67B-2BD902A0BE4C' as objectId,
+       now() as modified
```
Comment (labkey-martyp) [inline snprc_ehr/resources/queries/study/DemographicsHL7Deleted.sql]: Is this file needed?

**Review by labkey-martyp — APPROVED** [review]
Weird that some of these sql files are showing as binary in the list of file changes for this PR.  Looking at the file itself is not binary.

Anyway, just one comment, if you need the delete file.  Otherwise looks good.

File: snprc_ehr/resources/queries/study/DemographicsHL7Deleted.sql (hunk 1 above)
Comment (spamhurts) [reply snprc_ehr/resources/queries/study/DemographicsHL7Deleted.sql]: Good catch. I removed it from the PR - I created it when I was troubleshooting the missing schema issue. As for the binary SQL files, they were created in SSMS and saved with UTF-16LE encoding (Microsoft's default) - I converted them to UTF-8.
